cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/master-user-data.sh

```diff
@@ -11,6 +11,7 @@ MACHINE+={{ .Machine.ObjectMeta.Name }}
 ARCH=amd64
 swapoff -a
 # disable swap in fstab
+echo "127.0.0.1 $(hostname)"  >> /etc/hosts
```

Comment:
Do you only need it on the master node?

Reply:
I believe we need for all nodes..

Reply:
@sbueringer @jichenjc I only did it for master as the error was there only. I too feel it must be there in other nodes as well but did not find any instance to justify it. If you say I can add.

Reply:
yes, I think we can fix both as we anyway want to fix it:)

Reply:
Sure I will add it to the other user-data file as well.